### PR TITLE
docs: Add "Run your tests" section to .NET

### PR DESCRIPTION
**Pull request by HofmeisterAn**
### Proposed changes

The language-specific guide for .NET does not contain a section "Run your tests" as e.g. Java or Go has. This PR adds the missing section and explains how developers can use Docker to tests .NET applications. It extends the learnings from the previous sections and helps developers to integrate Docker in daily development.

### Related issues (optional)

\-

### CODE REVIEW

**Review by StefanScherer — APPROVED** [review]
Thanks @HofmeisterAn for the contribution.
LGTM

ping @craig-osterhout WDYT?

**Comment by craig-osterhout** [comment]
Thanks @HofmeisterAn and @StefanScherer

File: language/dotnet/run-tests.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+            .Build();
+
+        _dbContainer = new TestcontainersBuilder<TestcontainersContainer>()
+            .WithImage("postgres")
+            .WithNetwork(_network)
+            .WithNetworkAliases("db")
+            .WithVolumeMount("postgres-data", "/var/lib/postgresql/data")
+            .Build();
+
+        _appContainer = new TestcontainersBuilder<TestcontainersContainer>()
```
Comment (chrisbbe) [inline language/dotnet/run-tests.md:67]: `TestcontainersBuilder<>`  is disposable and thus should be disposed, how should one handle that?

Comment (HofmeisterAn) [reply language/dotnet/run-tests.md:67]: No, it is not necessary. The Resource Reaper will take care of it.

**Pull request merged**